common/djangoapps/student/views.py

```diff
@@ -693,11 +707,13 @@ def dashboard(request):
         'order_history_list': order_history_list,
         'courses_requirements_not_met': courses_requirements_not_met,
         'nav_hidden': True,
+        'course_programs': course_programs,
+        'program_marketing_url': urljoin(settings.MKTG_URLS.get('ROOT'), 'xseries' + '/marketing')
```

Comment:
just to make sure, "/marketing" will go away after #10304 is merged, and will be replaced with the value of program.marketing_slug - correct?